docs/index.md

```diff
@@ -60,8 +78,8 @@ other resources are listed on the [useful links] page.
 [useful links]: {{ site.baseurl }}/reference/useful-links/#tutorials
 
 ## Trophies
-As of February 2023, OSS-Fuzz has helped identify and fix over [8,900] vulnerabilities and [28,000] bugs across [850] projects.
+As of February 2023, OSS-Fuzz has helped identify and fix over [10,000] vulnerabilities and [31,000] bugs across [1,000] projects.
 
-[8,900]: https://bugs.chromium.org/p/oss-fuzz/issues/list?q=status%3AFixed%2CVerified%20Type%3DBug-Security&can=1
-[28,000]: https://bugs.chromium.org/p/oss-fuzz/issues/list?q=status%3AFixed%2CVerified%20Type%3DBug&can=1
-[850]: https://github.com/google/oss-fuzz/tree/master/projects
+[10,000]: https://bugs.chromium.org/p/oss-fuzz/issues/list?q=status%3AFixed%2CVerified%20Type%3DBug-Security&can=1
```

Comment:
Hmm...do we say this elsewhere?
The link here shows less than 10,000.
This link shows more than 10,000: https://bugs.chromium.org/p/oss-fuzz/issues/list?q=Type%3DBug-Security%20label%3Aclusterfuzz&can=1 (it's arguably more accurate